src/config/Config.zig

```diff
@@ -826,7 +826,7 @@ env: RepeatableStringMap = .{},
 link: RepeatableLink = .{},
 
 /// Enable URL matching. URLs are matched on hover with control (Linux) or
-/// super (macOS) pressed and open using the default system application for
+/// Command (macOS) pressed and open using the default system application for
```

Comment:
Why is it upper case?  That looks odd.
```suggestion
/// command (macOS) pressed and open using the default system application for
```